molecule/saas/playbook.yml

```diff
@@ -20,6 +20,11 @@
         path: "{{ lookup('env', 'LOCAL_MOUNT_PATH') }}"
     # change the default prefix to easily identify test containers / services
     hub_service_prefix: molecule_test_hub
+    jwt_privatekey: jwt-key
+    jwt_publickey: jwt-key.pub
+    jwt_local_key_path: "{{ (playbook_dir + '/../sample_keys/jwt') | realpath }}"
+    # Because docker in docker uses the host system instead of the actual remote
```

Comment:
SAAS scenario is not a docker in docker scenario. SAAS scenario spins up a hetzner VM just as deployment would happen  in a production environment